objective.py

```diff
@@ -19,20 +19,21 @@ def set_data(self, A, y):
         self.y = y
         S = np.sum(self.A, axis=1)
         self.c = self.get_c(S, self.delta)
-        self.reg = self.reg*self.get_reg_max(self.c)
+        self.reg_scaled = self.reg*self.get_reg_max(self.c)
 
     def compute(self, u):
         R = self.y - self.A @ u
         if self.data_fit == 'quad':
-            return .5 * R @ R + self.reg*(abs(np.diff(u)).sum())
+            return .5 * R @ R + self.reg_scaled*(abs(np.diff(u)).sum())
         else:
-            return self.huber(R, self.delta) + self.reg*(abs(np.diff(u)).sum())
+            return self.huber(R, self.delta) \
+                + self.reg_scaled*(abs(np.diff(u)).sum())
 
-    # def get_one_solution(self):
-    #     return np.zeros(self.A.shape[1])
+    def get_one_solution(self):
+        return np.zeros(self.A.shape[1])
 
     def to_dict(self):
-        return dict(A=self.A, reg=self.reg, y=self.y, c=self.c,
+        return dict(A=self.A, reg_scaled=self.reg_scaled, y=self.y, c=self.c,
```

Comment:
A simpler fix would have been to rename this `reg=self.reg_scaled` ^^

Reply:
thx! It's done!